src/main/java/fr/esgi/al5_2/Tayarim/services/AdministrateurService.java

```diff
@@ -0,0 +1,153 @@
+package fr.esgi.al5_2.Tayarim.services;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.*;
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
+import fr.esgi.al5_2.Tayarim.exceptions.*;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - fr.esgi.al5_2.Tayarim.exceptions.*.